benches/group/stabilizer_chain/mod.rs

```diff
@@ -2,22 +2,21 @@ pub mod selector;
 
 use criterion::{criterion_group, BenchmarkId, Criterion};
 const RANGE_OF_VALUES: [usize; 5] = [8, 10, 16, 20, 32];
+use rand::{Rng, SeedableRng};
 use stabchain::group::stabchain::base::selectors::DefaultSelector;
 use stabchain::group::stabchain::builder::random::parameters::RandomAlgoParameters;
 use stabchain::group::stabchain::builder::*;
 use stabchain::group::Group;
 use stabchain::perm::actions::SimpleApplication;
 
 use num::BigUint;
-use rand::SeedableRng;
 
 ///Macro for benchmarking a specific stabiliser chain strategy.
 macro_rules! bench_stabchain_impl {
     ($bencher: ident, $name:expr, $i:ident, $group:tt, $strat:expr) => {
         $bencher.bench_with_input(BenchmarkId::new($name, $i), $i, |b, i| {
             let g = $group(i);
-            let strat = $strat;
```

Comment:
Note that there is a slight difference here, if the cost of instantiating the strategy is non trivial the old code would have not measured it while the new one will. I think (not sure since I have not looked at the code in a bit) that for most of our strategies the creation cost is negligible and so we can keep this change but we should keep it in mind for the future.